apps/prairielearn/src/lib/question-render.ts

```diff
@@ -618,7 +615,7 @@ export async function getAndRenderVariant(
     const questionJson = JSON.stringify({
       questionFilePath: urls.calculationQuestionFileUrl,
       questionGeneratedFilePath: urls.calculationQuestionGeneratedFileUrl,
-      effectiveQuestionType,
+      effectiveQuestionType: 'Calculation',
```

Comment:
Nit: any reason why this wasn't changed in public/localscripts/question.js instead? I assume it's to avoid changing that code, and I'm ok if that's the case, just checking.